### PR TITLE
feat(web): enable websocket

### PR DESCRIPTION
**Pull request by alextran1502**
TODO
- [x] Add newly uploaded asset to new time bucket
- [x] Debouncing
- [ ] Album page
- [x] Photo page
- [x] Remove message about refreshing page

### CODE REVIEW

File: web/src/lib/stores/assets.store.ts [hunk 1]
```diff
@@ -164,6 +165,24 @@ export class AssetStore {
     return scrollTimeline ? delta : 0;
   }
 
+  addToBucket(asset: AssetResponseDto) {
+    const timeBucket = DateTime.fromISO(asset.fileCreatedAt).toUTC().startOf('month').toString();
+    const bucket = this.getBucketByDate(timeBucket);
+
+    if (!bucket) {
```
[the remark this answers is not in the document]
Comment (alextran1502) [reply web/src/lib/stores/assets.store.ts:172]: Yes, on my todo list

[the remark this answers is not in the document]
Comment (jrasm91) [reply web/src/lib/stores/assets.store.ts]: Cool sounds good!

File: web/src/lib/stores/assets.store.ts [hunk 2]
```diff
@@ -164,6 +167,49 @@ export class AssetStore {
     return scrollTimeline ? delta : 0;
   }
 
+  createBucket(bucketDate: string): AssetBucket {
+    const bucket = new AssetBucket();
+
+    bucket.bucketDate = bucketDate;
+    bucket.bucketHeight = THUMBNAIL_HEIGHT;
+    bucket.assets = [];
+    bucket.cancelToken = new AbortController();
```
Comment (jrasm91) [inline web/src/lib/stores/assets.store.ts:176]: This should be null.

File: web/src/lib/stores/assets.store.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import { api, AssetApiGetTimeBucketsRequest, AssetResponseDto } from '@api';
 import { writable } from 'svelte/store';
 import { handleError } from '../utils/handle-error';
+import { DateTime } from 'luxon';
+import lodash from 'lodash-es';
```
Comment (jrasm91) [inline web/src/lib/stores/assets.store.ts:5]: Can you only import denounce?

**Comment by jrasm91** [comment]
This is actually a bit tricky, because, even though we listen for the event, we should skip it if it doesn't belong to the current view. So, a new upload should not be added to the timeline if you are on the archive page.

**Pull request merged**